ts/src/test/test.ts

```diff
@@ -31,8 +33,7 @@ const ExchangeNotAvailable = ccxt.ExchangeNotAvailable;
 const OperationFailed = ccxt.OperationFailed;
 const OnMaintenance = ccxt.OnMaintenance;
 
-const [ processPath, , exchangeIdFromArgv = null, exchangeSymbol = undefined ] = process.argv.filter ((x) => !x.startsWith ('--'));
-const sanitizedSymnol = exchangeSymbol !== undefined && exchangeSymbol.includes ('/') ? exchangeSymbol : undefined;
```

Comment:
do you remind, why these lines were removed? they are used during development to test out specific method(s)

Reply:
Removing line 34 seems like a mistake. Removing line 35 was to allow passing test name as an argument to be tested